### PR TITLE
Import from hub docs

### PR DESCRIPTION
**Pull request by nataliaElv**
# Description

Closes #<issue_number>

**Type of change**

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (change restructuring the codebase without changing functionality)
- Improvement (change adding some improvement to an existing functionality)
- Documentation update

**How Has This Been Tested**

**Checklist**

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: argilla/docs/getting_started/quickstart.md [hunk 1]
```diff
@@ -59,6 +88,7 @@ pip install argilla
 ```
 
 ## Create your first dataset
```
Comment (burtenshaw) [inline argilla/docs/getting_started/quickstart.md]: We could actually just lose this section, and reference the guide on creating datasets in the SDK. 🤔

File: argilla/docs/getting_started/quickstart.md [hunk 2]
```diff
@@ -133,8 +163,9 @@ dataset.records.log(records=data, mapping={"text": "review"})
 🎉 You have successfully created your first dataset with Argilla. You can now access it in the Argilla UI and start annotating the records.
 
 ## Next steps
+
```
Comment (burtenshaw) [inline argilla/docs/getting_started/quickstart.md:170]: We could have a next step to upload data to the hub, so that users could use this feature without code? Or maybe this should be higher up. 🤔

File: argilla/docs/getting_started/quickstart.md [hunk 3]
```diff
@@ -48,7 +48,36 @@ In the sign in page:
...
+**Questions** are the feedback you want to collect, like labels, ratings, rankings, or text. If Argilla identified questions in your dataset that you don't want, you can eliminate them. You can also add questions of your own.
+
+!!! note
+    You will be able to modify some elements of the configuration of the dataset after it has been created from the Dataset Settings page e.g., the titles of fields and questions. Check all the settings you can modify in the [Update a dataset](../how_to_guides/dataset.md#update-a-dataset) section.
+
+When you're happy with the result, you'll need to give a name to your dataset, select a workspace and choose a split, if applicable. Then, Argilla will start importing the dataset in the background. Now you're all set up to start annotating!
+
+!!! note
+    If your dataset is bigger than 10k records, at this stage Argilla will only import the first 10k. You can import the rest of the records at any point using the Python SDK.
```
Comment (burtenshaw) [inline argilla/docs/getting_started/quickstart.md]:
```suggestion
    Argilla will only import the first 10k rows of a dataset. If your dataset is larger, you can import the rest of the records at any point using the Python SDK.
```

**Review by burtenshaw — COMMENTED** [review]
Looks pretty good. 

This page is starting to get a lot of boxes. i.e. `!!! note` . When you read it on the frontend it's quite hard to follow. I would skim the page and see if some can be converted to in line text.

**Comment by nataliaElv** [comment]
> Looks pretty good.
> 
> This page is starting to get a lot of boxes. i.e. `!!! note` . When you read it on the frontend it's quite hard to follow. I would skim the page and see if some can be converted to in line text.

Solved!

**Review by sdiazlor — COMMENTED** [review]
Just some comments as a person with little knowledge of this feature and is taking a first look.

- As the quickstart, I'd say to change the title of "Import a dataset from the Hub" to something more direct as it can be confusing for someone who doesn't know about Argilla. I'd say to use here "Create your first dataset", and that section remove it, adding just a note as Ben suggested and including SDK related stuff in a "Using the python SDK", similarly to "Deploy with Docker" where only it is mentioned and redirected. Or even combining the "importing long dataset" issue and the sdk further configuration/creation mention.
- I'll move the image after Fields and Questions so people can understand what they see.
- Not sure but regarding the Dataset Settings update, isn't it after giving it a name? I'd use a note or move it after that if that's the case.
- About workspaces, I'd keep the "About workspaces" as it seems it must be manually added and it's not automatically inferred?
- If this is the unique doc related to importing from the UI, it would be worthy to mention others like what about vectors or metadata, as those already knowing Argilla will have to check.

File: argilla/CHANGELOG.md [hunk 4]
```diff
@@ -27,6 +27,10 @@ These are the section headers that we use:
 
 - Removed name sanitizing for dataset settings names. This may cause issues with old server versions. Especially when working with `from_hub`. ([#5574](https://github.com/argilla-io/argilla/pull/5574))
 
+### Removed
+
+- Removed name sanitizing for dataset settings names. This may cause issues with old server versions. Especially when working with `from_hub`. ([#5574](https://github.com/argilla-io/argilla/pull/5574))
+
```
Comment (frascuchon) [inline argilla/CHANGELOG.md]:
```suggestion
```
